### ISSUE TITLE
Text is dessapearing when I try to type

### ISSUE THREAD

**Issue by Gerald17** [esbenp/react-native-clean-form#48]
Text automatically is deleted when I type something.
I'm working on android, any ideas?

**Comment by crjackso** [comment]
I'm having this same issue.  @Gerald17, did you find a solution?

**Comment by Gerald17** [comment]
Hi @crjackso 

Not really for this. Its a weird issue but I replace this form with this https://github.com/halilb/react-native-textinput-effects/blob/master/README.md  it has some cool effects for inputs and you can use state to handle data

**Comment by markusguenther** [comment]
Sorry work and my little kids take a lot of time actually ... but try to look on the issues.

**Comment by Gerald17** [comment]
Don't worry, still is really good form. I'll try to fix it too

**Comment by sejsworld** [comment]
I have the same issue, text disappers all the time, as well as selections from select

**Comment by clay-morgan** [comment]
Maybe from not wiring up the form reducer when using redux-form. I do this sometimes.

**Comment by infostreams** [comment]
I can confirm that this behaviour is caused by not wiring up `redux-form` correctly, and basically not using the provided reducer in your redux store. 

An example of how to wire up `redux-form` correctly is in the example, specifically [here](https://github.com/esbenp/react-native-clean-form/blob/master/example/src/index.js#L10). You can install the non-immutable reducer as follows:

```es6
import { reducer as form } from 'redux-form'

const reducer = combineReducers({ form })
const store = createStore(reducer, fromJS({}))
```

If you use `redux-form` for your Form, it will use the information it finds in Redux to populate the form fields. If you didn't install the reducer, then the state in Redux will stay perpetually empty, and this empty state will be transmitted to your form fields, which will manifest itself by disappearing the text as you type it. Mystery solved 😉 Hope this helps someone.

**Comment by infostreams** [comment]
I actually think this issue can be closed...

**Comment by The-Code-Monkey** [comment]
yeah the reason text was disappearing is because you didnt have the reducer form setup so redux had nowhere to put the data
